__tests__/app.test.js

```diff
@@ -36,6 +36,7 @@ describe('/api/reviews/:review_id', () => {
       category: 'euro game',
       created_at: '2021-01-18T10:00:20.514Z',
       votes: 1,
+      comment_count: '1',
```

Comment:
I would have written this as an extra test, but done this one as expect objectEqual